package.json

```diff
@@ -94,13 +94,9 @@
     "resolutions": {
         "//": {
             "http-cache-semantics": "Pinned to address security vulnerability",
-            "jsonwebtoken": "Pinned to address security vulnerability",
-            "semver": "Pinned to address security vulnerability",
-            "systeminformation": "Pinned to address security vulnerability"
+            "semver": "Pinned to address security vulnerability"
         },
         "http-cache-semantics": "4.1.1",
```

Comment:
Still needed for `karma-sauce-launcher-fix-firefox`; I will push a new version of that and then we can get rid of this.

Reply:
Actually, that's more trouble than it's worth. We should just move off of Karma eventually anyway.